cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java

```diff
@@ -96,14 +97,25 @@ public void tearDown() throws Exception {
     // TODO: check metrics, streams, etc.
   }
 
-  protected TestManager getTestManager() {
+  protected ClientConfig getModifiableClientConfig(ClientConfig initialClientConfig, Id.Namespace namespace) {
+    ClientConfig newClientConfig = new ClientConfig.Builder(initialClientConfig).build();
+    newClientConfig.setNamespace(namespace);
+    return newClientConfig;
+  }
+
+  protected TestManager getTestManager(Id.Namespace namespace) {
```

Comment:
Call it `createTestManager`. For method that create new instance, call it `createXXX`.